ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -342,9 +342,21 @@ tests:
       TEST_TIMEOUT: "30"
     test:
     - ref: cucushift-helper-service-build-info
-    - ref: idp-htpasswd
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: e2e-aws-ipi-ovn-hypershift-install-p2
```

Comment:
Looks like there are sort issue ?

Reply:
You mean to put between `e2e-aws-ipi-ovn-fips-p3-destructive` and `e2e-aws-ipi-ovn-ingress-nlb-p1` right?

Reply:
Yes, as `e2e-aws-ipi-ovn-fips-xxx` sorts before `e2e-aws-ipi-ovn-hypershift-xxx` ?

Reply:
@liangxia could you help to review again? thanks